From 7472764fe19cb3eea77f21cc2859859b8ac62085 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Mon, 14 Aug 2017 23:36:36 +0200
Subject: [PATCH] configure.ac: do not force -O2

The user may want to provide its own set of optimization flags, and
therefore forcing -O2 should not be done.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Submitted-upstream: https://github.com/libspatialindex/libspatialindex/pull/101
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index dc5e49f..2e2aa68 100644
--- a/configure.ac
+++ b/configure.ac
@@ -53,7 +53,7 @@ if test "x$enable_debug" = "xyes"; then
 	CXXFLAGS="$CXXFLAGS -g -DDEBUG"
 	AC_MSG_RESULT(checking wether debug information is enabled... yes)
 else
-	CXXFLAGS="$CXXFLAGS -O2 -DNDEBUG"
+	CXXFLAGS="$CXXFLAGS -DNDEBUG"
 	AC_MSG_RESULT(checking wether debug information is enabled... no)
 fi
 
-- 
2.9.4

